When covers cannot be scraped, scraping panics

panic: runtime error: index out of range [0] with length 0

goroutine 222 [running]:
github.com/cld9x/xbvr/pkg/xbvr.SceneCreateUpdateFromExternal(0xc0018efc70, 0xc00692dac0, 0x12, 0xc006d6a6f9, 0x5, 0x124974b, 0x2, 0x0, 0x0, 0x1261148, ...)
        /home/jrebey/workspaces/xbvr/pkg/xbvr/model_scene.go:155 +0x101b
github.com/cld9x/xbvr/pkg/xbvr.Scrape()
        /home/jrebey/workspaces/xbvr/pkg/xbvr/task_content.go:108 +0xc7e
created by github.com/cld9x/xbvr/pkg/xbvr.TaskResource.scrape
        /home/jrebey/workspaces/xbvr/pkg/xbvr/api_tasks.go:60 +0x35
exit status 2

This particular issue is caused by:
https://ddfnetworkvr.com/videos/my+girlfriend+s+a+nympho/21687

--- a/pkg/xbvr/model_scene.go
+++ b/pkg/xbvr/model_scene.go
@@ -149,7 +149,9 @@ func SceneCreateUpdateFromExternal(db *gorm.DB, ext scrape.ScrapedScene) error {
 	o.Duration = ext.Duration
 	o.Synopsis = ext.Synopsis
 	o.ReleaseDateText = ext.Released
-	o.CoverURL = ext.Covers[0]
+	if ext.Covers != nil {
+		o.CoverURL = ext.Covers[0]	
+	}
 	o.SceneURL = ext.HomepageURL
 
 	if ext.Released != "" {

